dist/stat_data.py

```diff
@@ -264,12 +264,15 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_lookaside_cursor_wait_internal', 'cache overflow cursor internal thread wait time (usecs)'),
     CacheStat('cache_lookaside_entries', 'cache overflow table entries', 'no_clear,no_scale'),
     CacheStat('cache_lookaside_insert', 'cache overflow table insert calls'),
+    CacheStat('cache_lookaside_prepare_insert', 'prepared updates added to cache overflow'),
     CacheStat('cache_lookaside_remove', 'cache overflow table remove calls'),
     CacheStat('cache_lookaside_score', 'cache overflow score', 'no_clear,no_scale'),
     CacheStat('cache_overhead', 'percentage overhead', 'no_clear,no_scale'),
     CacheStat('cache_pages_dirty', 'tracked dirty pages in the cache', 'no_clear,no_scale'),
     CacheStat('cache_pages_inuse', 'pages currently held in the cache', 'no_clear,no_scale'),
     CacheStat('cache_pages_requested', 'pages requested from the cache'),
+    CacheStat('cache_prepared_updates_active', 'prepared updates active in the cache'),
+    CacheStat('cache_prepared_updates_resolved', 'prepared updates resolved in the cache'),
```

Comment:
A few comments:

* I'd probably expect these two, and maybe the lookaside stat, to be in the `TxnStat` section.  Since prepared updates don't pin pages in cache, that makes more sense to me than grouping them with the cache stats.
* The "updates active" only counts updates added to cache: it is never decremented.  Is that expected?
* As far as I can tell, "resolved" doesn't include rolled back.  Either add rollbacks in here or add a new stat.

What I think triage engineers would want to know is "how many updates are currently in the prepared state" -- if we can have a single stat for that, which goes up and down, that would probably be better than having to combine multiple stats to figure it out.